src/foremast/utils/lookups.py

```diff
@@ -242,9 +243,9 @@ def json(self, branch='master', filename=''):
         file_contents = self.get(branch=branch, filename=filename)
 
         try:
-            json_dict = json.loads(file_contents)
+            json_dict = yaml.load(file_contents)
         # TODO: Use json.JSONDecodeError when Python 3.4 has been deprecated
```

Comment:
Should we remove this TODO since you are not using json lib anymore?

Reply:
Probably a good idea. :+1: